recipes/sip/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [ `uname` == Darwin ]; then
+    pythonw configure.py --sysroot=$PREFIX
```

Comment:
BTW, why is it important to use `pythonw` here instead of `python`?

Not saying its a problem -- I'm selfishly hoping to learn something. To be honest, I don't understand the difference.  I can see that `pythonw` merely calls `python.app/Contents/MacOS/python` ... what is `python.app` for?